src/main/java/paimon/commands/Command.java

```diff
@@ -0,0 +1,30 @@
+package paimon.commands;
+
+import paimon.tasklist.TaskList;
+import paimon.ui.UI;
+
+/**
+ * Abstract class for all commands.
+ * A command is an action that can be executed with a tasklist and ui given
+ * a user input.
+ */
+public abstract class Command {
+    /**
+     * Execute the command with current tasklist and ui
+     * 
+     * @param t tasklist involved
```

Comment:
@param should end with a full stop
```suggestion
     * @param t tasklist involved.
```